.github/ISSUE_TEMPLATE/pixel_issue.md

```diff
@@ -0,0 +1,34 @@
+---
+name: Pixel Data issue
+about: For issues related to Pixel Data or one of the bulk data handlers
+title: ''
+labels: pixel-data
```

Comment:
Oh, I thought it might be good to have a label for all the pixel data related issues.

Reply:
Good idea!

Reply:
I like that too